### PR TITLE
[HWORKS-1232] Adding Jenkins file for building hive docker image

### PR DESCRIPTION
**Pull request by Jacarte**

### CODE REVIEW

File: dockerfiles/Dockerfile [hunk 1]
```diff
@@ -12,6 +12,7 @@ ARG METASERVER_ID="1505"
 ARG PROMETHEUS_EXPORTER_VERSION="0.12.0"
 ARG MYSQL_CONNECTOR_VERSION="8.0.21"
 ARG HUDI_BUNDLE_VERSION="0.12.3.0"
+ARG TEZ_VERSION="0.9.1.5"
```
Comment (maismail) [inline dockerfiles/Dockerfile:15]: you need to rebase since we just merged this PR

File: dockerfiles/HIVE_VERSION [hunk 2]
```diff
@@ -0,0 +1 @@
+3.0.0.13.3
```
Comment (maismail) [inline dockerfiles/HIVE_VERSION:1]: we don't need this file

File: dockerfiles/Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+
+node("local") {
+    stage('Clone repository') {
+      checkout scm
+    }
+
+    stage('Build and push image(s)') {
+      withCredentials([usernamePassword(credentialsId: 'a0770738-4ef3-4acc-a6ba-097ee6c85b44', passwordVariable: 'PASSWORD', usernameVariable: 'USERNAME')]) {
+        version = readFile "${env.WORKSPACE}/dockerfiles/HIVE_VERSION"
```
Comment (maismail) [inline dockerfiles/Jenkinsfile:13]: we should get the version from the pom file similar to what we do here 
https://github.com/logicalclocks/clusterj-onlinefs/blob/master/Jenkinsfile#L18

**Review by maismail — CHANGES_REQUESTED** [review]

File: dockerfiles/Dockerfile (hunk 1 above)
Comment (Jacarte) [reply dockerfiles/Dockerfile]: Fixed

**Review by maismail — APPROVED** [review]

**Pull request merged**